deployment/rpm/about.md

```diff
@@ -128,7 +128,7 @@ $ rpmbuild -bs hello.spec
 Then you can run:
 
 ```bash
-$ mock -r epel-7-i386.cfg hello.src.rpm
+$ mock -r /etc/mock/epel-6-i386.cfg ~/rpmbuild/SRPMS/hello-2.10-1.fc27.src.rpm
```

Comment:
Why `epel6`? Also, why `i386` and not `x86_64`? (I just wonder if there are some reasons behind it?)
Also I'd say it's really not needed to use full path.

Reply:
This should absolutely be for latest fedora and el, not el6...

Reply:
> for latest fedora and el, not el6...

Actually, as long as this works *on Fedora* it does not matter.

Reply:
Ok, let's assume the Contributor has tested this and the values do not matter in the end as it's just an example.